src/nav/apm.yml

```diff
@@ -3,11 +3,17 @@ path: /docs/apm
 pages:
   - title: Introduction to APM
     path: /introduction-apm
-  - title: APM best practice guide
-    path: /docs/new-relic-solutions/best-practices-guides/full-stack-observability/apm-best-practices-guide
+  - title: Guides
+    pages:
+    - title: APM best practice guide
+      path: /docs/new-relic-solutions/best-practices-guides/full-stack-observability/apm-best-practices-guide
+    - title: 'Measure user satisfaction with Apdex'
+      path: /docs/apm/new-relic-apm/apdex/apdex-measure-user-satisfaction
   - title: APM data security
     path: /docs/apm/new-relic-apm/getting-started/apm-agent-data-security
-  - title: Install and configure APM agents
+  - title: Triage and run diagnostics in ten clicks
```

Comment:
we say eight clicks in title. (and again, I do wonder if that's too clickbaity for docs